tools/src/main/scala/scala/scalanative/linker/Reach.scala

```diff
@@ -358,6 +358,7 @@ class Reach(config: build.Config, entries: Seq[Global], loader: ClassLoader) {
 
   def reachVar(defn: Defn.Var): Unit = {
     val Defn.Var(attrs, name, ty, rhs) = defn
+    import defn.pos
```

Comment:
The fact that `defn.pos` is itself an `implicit` here is an accident and we should not rely on it. It is too confusing. Consider instead:
```suggestion
    implicit val pos = defn.pos
```
Same later in this file.